bin/wgif

```diff
@@ -1,4 +1,5 @@
 #!/usr/bin/env ruby
 
+$: << File.join(File.expand_path(__dir__), "..", "lib")
```

Comment:
Two things here: 1) `__dir__` is Ruby 2+ only, 2) I think the `".."` is incorrect (although I can never decipher these :unamused:):

```
irb(main):001:0> RUBY_VERSION
=> "2.2.2"
irb(main):002:0> File.join(File.expand_path(__dir__), "..", "lib")
=> "/Users/ecm/Projects/ruby/wgif/../lib"
```

If you want to support my arbitrarily selected assortment of Ruby versions, I think you need `File.join(File.expand_path('..', __FILE__), 'lib')`:

```
irb(main):001:0> RUBY_VERSION
=> "2.2.2"
irb(main):002:0> File.join(File.expand_path('..', __FILE__), 'lib')
=> "/Users/ecm/Projects/ruby/wgif/lib"
```

```
irb(main):001:0> RUBY_VERSION
=> "2.0.0"
irb(main):002:0> File.join(File.expand_path('..', __FILE__), 'lib')
=> "/Users/ecm/Projects/ruby/wgif/lib"
```

```
irb(main):001:0> RUBY_VERSION
=> "1.9.3"
irb(main):002:0> File.join(File.expand_path('..', __FILE__), 'lib')
=> "/Users/ecm/Projects/ruby/wgif/lib"
```